app/helpers/date_time_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module DateTimeHelper
+  def format_time(datetime)
+    datetime.strftime("%-l:%M%P")
+  end
+
+  def scheduled_date_placeholder
+    publish_time ||= Time.current.tomorrow.change(hour: 9)
+    {
+      day: params.dig(:schedule, :date, :day) || publish_time.day,
```

Comment:
Helpers can be called from any view, so to make this class reusable it would be better to pass in date and time, they can have default values of `nil` so that the conditionals still work.

e.g. `def default_date_time(date=nil, time=nil)`

Reply:
That would make it easier to test, too!

Reply:
Hmm that becomes a bit of a pain to use in schedule.html:

```erb
        items: [
          {
            name: "day",
            width: 2,
            value: scheduled_date_placeholder(nil, nil, params.dig(:schedule, :date, :day), nil)[:day]
          },
```

I could do this instead:

```erb
        items: [
          {
            name: "day",
            width: 2,
            value: params.dig(:schedule, :date, :day) || scheduled_date_placeholder[:day]
          },
```

(...and remove the `||` logic from `scheduled_date_placeholder`). It's quite a front-end-y bit of logic anyway.

Reply:
Or you can pass in the whole date.